ci/poll/api.go

```diff
@@ -0,0 +1,56 @@
+/*
+|    Protect your secrets, protect your sensitive data.
+:    Explore VMware Secrets Manager docs at https://vsecm.com/
+</
+<>/  keep your secrets… secret
+>/
+<>/' Copyright 2023–present VMware, Inc.
+>/'  SPDX-License-Identifier: BSD-2-Clause
+*/
+
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"time"
+)
+
+const (
+	githubAPIURL = "https://api.github.com/repos/vmware-tanzu/secrets-manager/git/refs/heads/main"
+	pollInterval = 15 * time.Minute
+)
+
+type GitHubResponse struct {
+	Object struct {
+		SHA string `json:"sha"`
+	} `json:"object"`
+}
+
+func getLatestCommitHash() (string, error) {
+	resp, err := http.Get(githubAPIURL)
```

Comment:
(Minor) I believe we should also as a check to see if the resp.StatusCode is not other than the expected one.

Reply:
Makes sense… I’ll add that in a follow-up PR.